content/docs/2.8/scalers/azure-pipelines.md

```diff
@@ -60,7 +66,24 @@ The URL should be similar to `https://dev.azure.com/{organization}/_settings/age
 
 Finally, it is also possible get the pool ID from the response of a HTTP request by calling the `https://dev.azure.com/{organizationName}/_apis/distributedtask/pools?poolname={agentPoolName}` endpoint in the key `value[0].id`.
 
-### Example
+### Supporting Demands in Agents ###
+
+Parent Template: Azure Devops is able to determine which agents can match any job its is waiting for. If you specify a parent template then KEDA will further interrogate the job request to determine if the parent is able to fulfil the job. If the parent is able to complete the job it scales up the ScaledJob to fulfil it. The parent template is generally offline must stays in the Pool's Agent list.
+
+Demand List: KEDA will determine which agents can fulfil the job based on the demands provided. The demands are provided as a comma separated list and must be a subset of the actual capabilities of the agent. (Note: Agent.Version is ignored)
```

Comment:
```suggestion
TODO: Add some intro here before listing approaches:

- **Using Parent:** Azure DevOps is able to determine which agents can match any job it is waiting for. If you specify a parent template then KEDA will further interrogate the job request to determine if the parent is able to fulfill the job. If the parent is able to complete the job it scales the workload fulfill the request. The parent template that is generally offline must stay in the Pool's Agent list.

- **Using demands:** KEDA will determine which agents can fulfill the job based on the demands provided. The demands are provided as a comma-separated list and must be a subset of the actual capabilities of the agent. (For example `someexamplehere`. Note: `Agent.Version` is ignored)
```

Can we add a link to the official parent & demands doc per entry?

Reply:
I can add links for demands, but parent is an internal concept to support this model, its not Microsoft's